.yarnrc

```diff
@@ -0,0 +1 @@
+--ignore-engines true
```

Comment:
Not strictly related to testing but this suppresses some warnings and is included in the yeoman template for vscode extensions that's recommended by the vscode team.

Reply:
👍  great, I didn't like that warning, but didn't know how to suppress it